gw_eccentricity/eccDefinition.py

```diff
@@ -1597,6 +1814,33 @@ def measure_ecc(self, tref_in=None, fref_in=None):
         # return measured eccentricity, mean anomaly and reference time or
         # frequency where these are measured.
         return self.make_return_dict_for_eccentricity_and_mean_anomaly()
+    
+    def build_omega_gw_extrema_interpolants(self):
```

Comment:
Needs a docstring explaining what it's doing and why.

Reply:
Done